File: 10 - Python/Exercícios/03 - Estruturas Repetitivas/For/Exercício 02.py
#Exercício - Soma Ímpares

soma = 0
troca = 0

print("Digite dois números:")
x = int(input(""))
y = int(input(""))

if x > y:
    troca = x
    x = y
    y = troca

for i in range(x+1, y):
    if i % 2 != 0:
        soma = soma + i
    
print("Soma dos ímpares =", soma)

File: 07 - C++/Exercícios/03 - Estruturas Repetitivas/For/Exercício 02.cpp
//Exercício - Soma ímpares

#include <iostream>

using namespace std;

int main(){

    int i, x, y, soma, troca;

    cout << "Digite dois numeros:\n";
    cin >> x;
    cin >> y;

    if(x > y){
        troca = x;
        x = y;
        y = troca;
    }

    soma = 0;

    for(i = x+1; i <= y-1; i++){

        if(i % 2 != 0){
            soma = soma + i;
        }
    }

    cout << "Soma dos impares = " << soma << endl;

    return 0;

}